### PR TITLE
JA-translation-fledge-experiment

### PR DESCRIPTION
**Pull request by mattletford**
@agektmr JA translation of fledge experiment

### CODE REVIEW

**Comment by mattletford** [comment]
> This is wrong. Why did you copy / paste from older version of another article? Here's the source doc expected to be converted from: https://docs.google.com/document/d/13awsSXNqncHVoX4KD9-Bnbh3_qENlSveeFxAhXJFO54/edit?resourcekey=0-xMucu0lRrTA4Oe_uy3MqFw

Thank you, I've updated to the new doc

File: site/ja/docs/privacy-sandbox/fledge-experiment/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    - [FLEDGE API デベロッパーガイド](/blog/fledge-api/)では、API メソッドとパラメータを参照できます。
+
+    - [FLEDGE ワークレットのデバッグ](/blog/fledge-api/#debug-fledge-worklets)では、Chrome DevTools を使って FLEDGE の入札とオークションのコードをデバッグする方法について説明しています。
+
+    - [デベロッパーガイド](/blog/fledge-api/#what-features-are-supported-behind-these-feature-flags-in-the-latest-version-of-chrome)では、最新バージョンの Chrome でサポートされている機能について説明しています。[API の解説](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#summary)では、機能のサポートと制約についてさらに詳しく説明しています。
+
+## サポートを受ける
+
+API のテストをうまく進められない場合はご連絡ください。
+
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fledge-experiment/index.md:41]:
```suggestion

{% Aside %}
API のテストをうまく進められない場合はご連絡ください。
{% endAside %}

```

File: site/ja/docs/privacy-sandbox/fledge-experiment/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'FLEDGE: テストとディスカッションへの参加'
+subhead: リマーケティングに活用できるソリューションです。このソリューションは、第三者がサイト間のユーザーの閲覧行動を追跡できないように設計されています。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fledge-experiment/index.md:4]:
```suggestion
subhead: FLEDGE はプライバシー サンドボックスの提案で、リマーケティングとカスタム オーディエンスに利用することができますが、第三者がサイト間のユーザーの閲覧行動を追跡できないように設計されています。
```

File: site/ja/docs/privacy-sandbox/fledge-experiment/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+description: FLEDGE はリマーケティングに活用可能でありながら、第三者がサイト間のユーザーの閲覧行動を追跡できないように設計されています。この API を使用すると、ユーザーが以前にアクセスした Web サイトが提供する関連広告を選択するためのブラウザーによるデバイス上の「オークション」を実現できます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fledge-experiment/index.md:5]:
```suggestion
description: FLEDGE はプライバシー サンドボックスの提案で、リマーケティングとカスタム オーディエンスに利用することができますが、第三者がサイト間のユーザーの閲覧行動を追跡できないように設計されています。この API は、ユーザーが以前にアクセスした Web サイトが提供する関連広告を選択するための、ブラウザーによるデバイス上の「オークション」を実現します。
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by mattletford** [comment]
@agektmr I've applied the suggested changes

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by mattletford** [comment]
@agektmr I've applied the suggested changes

File: site/ja/docs/privacy-sandbox/fledge-experiment/index.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+- プライバシー サンドボックスの[フィードバックフォーム](/docs/privacy-sandbox/feedback/#feedback-form)を使用して、公開フォーラムではなく Chrome チーム宛に非公開でフィードバックを送信できます。
+
+- [プライバシーサンドボックスのフィードバック](/docs/privacy-sandbox/feedback/#fledge-api)では、他の種類のフィードバックを送信する方法や、プライバシーサンドボックスの提案に関するディスカッションに参加する方法について説明しています。
+
+## 最新情報を入手する
+
+- この API のステータス変更について通知を受け取るには、[デベロッパー向けのメーリングリスト](https://groups.google.com/u/3/a/chromium.org/g/fledge-api-announce)にご参加ください。
+
+- この API に関する進行中のディスカッションすべてを常にチェックするには、[GitHub の提案ページ](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)で ボタンをクリックしてください。なお、この操作を行うには GitHub > アカウントが必要です。お持ちでない場合は [GitHub アカウントを作成](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)してください。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fledge-experiment/index.md:88]:
```suggestion
- この API に関する進行中のディスカッションすべてを常にチェックするには、[GitHub の提案ページ](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)で **Watch** ボタンをクリックしてください。なお、この操作を行うには GitHub > アカウントが必要です。お持ちでない場合は [GitHub アカウントを作成](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)してください。
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by mattletford** [comment]
@agektmr I've applied the suggested changes

**Pull request merged**